### DISCUSSION TITLE
Projects using motion

### DISCUSSION THREAD

**Discussion by jdodgen** [Motion-Project/motion#1825] [General]
Is there a list of projects that use motion or motion plus. If so I would like to add [https://github.com/jdodgen/basicNVR] it is a system i have used for years and now open sourcing it it.

**Comment by Mr-Dave** [comment]
There is a section under the [wiki](https://github.com/Motion-Project/motion/wiki/Related-Projects).  If you want to give me a short sentence, I can add it into the page.

**Reply by jdodgen** [reply]
basicNVR a simplified network video recorder
